examples/visual-tests/amp-by-example/LICENSE

```diff
@@ -181,7 +178,7 @@
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "{}"
+      boilerplate notice, with the fields enclosed by brackets "[]"
```

Comment:
I think this section was (and still is) misleading. Per https://www.apache.org/licenses/LICENSE-2.0, the text that follows this sentence should read...
```
Copyright [yyyy] [name of copyright owner]
```
... and that isn't the case here. I think we should remove the appendix.

Reply:
Adding @cramforce for comment.

Reply:
Perhaps we can address this in another PR and get this typo fix in?